Form-Controls/index.html

```diff
@@ -4,25 +4,53 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <title>My form exercise</title>
-  <meta name="description" content="">
+  <meta name="description" content="T-shirt order form">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
-  <header>
+  <header role="banner">
```

Comment:
I like that you're thinking about roles, but you don't need to add the role banner to a header - it carries this role already. 

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/banner_role